ecommerce/src/main/kotlin/io/hhplus/ecommerce/product/application/usecase/ProductRankingQueryUseCase.kt

```
@@ -0,0 +1,167 @@
+            )
+        }
+    }
+
+    /**
+     * 현재 날짜의 연도-주차 문자열 반환
+     */
+    private fun getYearWeek(date: LocalDate): String {
+        val weekFields = WeekFields.of(Locale.KOREA)
+        val year = date.get(weekFields.weekBasedYear())
+        val week = date.get(weekFields.weekOfWeekBasedYear())
+        return String.format("%d%02d", year, week)
```

detekt (1.23.8)

[warning] String.format("%d%02d", year, week) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)